src/foam/nanos/theme/Theme.js

```diff
@@ -100,6 +100,23 @@ foam.CLASS({
       name: 'appName',
       section: 'infoSection',
     },
+    {
+      class: 'String',
+      name: 'loginDAO'
+    },
+    {
+      class: 'String',
+      name: 'loginImage'
+    },
+    {
+      class: 'String',
+      name: 'defaultGroup'
```

Comment:
Where is this used?

Reply:
SignUp.js uses this for setting the group on the newly created User.

Reply:
This should be done elsewhere - rules based on spid and urls.  just default the code to 'sme'.  This also will have to change based on Ablii apps.

Reply:
Sounds good.